packages/pharos-site/static/guidelines/text-input.docs.mdx

```diff
@@ -69,35 +66,51 @@ import BestPractices from '../../src/components/statics/BestPractices.tsx';
 <PageSection title="Variants">
   <PageSection subSectionLevel={1} title="Text">
     <p>
-      The text variant should be used when the user needs to enter a single-line value and there
-      isn't a more specific input type available for collecting that value.
+      The text variant is a single-line text field for generic text, used when there isn't a more
+      specific input type available for collecting a value.
     </p>
+    <Canvas>
+      <PharosTextInput type="text" placeholder="text"></PharosTextInput>
+    </Canvas>
   </PageSection>
   <PageSection subSectionLevel={1} title="Password">
     <p>
-      A password input is a single-line text field whose value is obscured. Use it when you need to
+      A password input is a single-line text field whose value is obscured, used when you need to
       protect the user's content from being read.
     </p>
+    <Canvas>
+      <PharosTextInput type="password" value="password"></PharosTextInput>
+    </Canvas>
   </PageSection>
   <PageSection subSectionLevel={1} title="Email">
     <p>
-      A search input allows the user to enter search queries. Line-breaks are automatically removed
-      from the input value. They may include a delete or "clear" icon in supporting browsers that
-      can be used to clear the field.
+      The email variant provides validation to ensure proper formatting of an email address. Use the
```

Comment:
I would avoid using "variant" here unless we mean a `variant` property or a set of polymorphic components—`type` feels a bit more appropriate because it's the name of the HTML attribute involved, which you might want to use in the `number` and `tel` sections below as well:

```suggestion
      The email type provides validation to ensure proper formatting of an email address. Use the
```